mythtv/programs/mythfrontend/themechooser.cpp

```diff
@@ -167,55 +167,32 @@ void ThemeChooser::Load(void)
         }
     }
 
-    // MYTH_SOURCE_VERSION - examples v29-pre-574-g92517f5, v29-Pre, v29.1-21-ge26a33c
-    QString MythVersion(GetMythSourceVersion());
-    static const QRegularExpression trunkver{"\\Av[0-9]+-pre.*\\z", QRegularExpression::CaseInsensitiveOption};
-    static const QRegularExpression validver{
-        "\\Av[0-9]+.*\\z", QRegularExpression::CaseInsensitiveOption};
-
-    auto match = validver.match(MythVersion);
-    if (!match.hasMatch())
-    {
-        LOG(VB_GENERAL, LOG_ERR, QString("Invalid MythTV version %1, will use themes from trunk").arg(MythVersion));
-        MythVersion = "trunk";
-    }
-    match = trunkver.match(MythVersion);
-    if (match.hasMatch())
-        MythVersion = "trunk";
+    uint8_t major { 0 };
+    uint8_t minor { 0 };
+    bool    devel { false };
+    bool    parsed = ParseMythSourceVersion(devel, major, minor);
 
-    if (MythVersion == "trunk")
+    if (!parsed || devel)
     {
-        LoadVersion(MythVersion, themesSeen, true);
-        LOG(VB_GUI, LOG_INFO, QString("Loading themes for %1").arg(MythVersion));
+        if (!parsed)
+            LOG(VB_GENERAL, LOG_ERR,
+                QString("Invalid MythTV version %1, will use themes from trunk")
+                .arg(GetMythSourceVersion()));
+        LOG(VB_GUI, LOG_INFO, QString("Loading themes for trunk"));
+        LoadVersion("trunk", themesSeen, true);
     }
```

Comment:
There must be lots of people now who think that a trunk can be either a suitcase or the storage place at the back of car, and how you possibly store themes in there.
I think it is more clear to use "development" and "release" to indicate which collection of themes is being used.

Reply:
Good point. I can't change the actual filenames without changing the themes backend service, but I can change the messages.

I am currently working to migrate the backend services to a new server, so I will look to add an a new 'release' directory.   Once done, I can circle back and change the actual filenames too. The old 'trunk' directory will need to hang around forever to support older clients. (I can't just add an alias from 'release' to 'trunk' because its out of my control. Those files are served up by OSUOSL.)